data-access/mandataris.ts

```diff
@@ -615,11 +620,13 @@ export async function bulkSetPublicationStatusEffectief(
     DELETE {
       GRAPH ?graph {
         ?mandataris lmb:hasPublicationStatus ?status .
+        ?mandataris lmb:effectiefAt ?effectiefAt .
```

Comment:
Normally this would be no value but better save than sorry

Reply:
yup!